build.sbt

```diff
@@ -5,7 +5,7 @@ organization := "org.zalando"
 
 scalaVersion := "2.11.8"
 
-libraryDependencies += "org.apache.spark" %% "spark-sql" % "2.0.1"  % Provided
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "2.4.3"  % Provided
```

Comment:
this is outdated in the new master. That's my fault for replying so late, sorry again.